docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -69,10 +74,19 @@ metadata:
   name: ovn-copy-data
   annotations:
     openshift.io/scc: anyuid
+ifeval::["{OpenStackPreviousInstaller}" != "director_operator"]
     k8s.v1.cni.cncf.io/networks: internalapi
```

Comment:
Line 78 is specific to OSPdO?

Reply:
yes, meaning that line: 
  '[{"name": "internalapi-static", "namespace": ${OSPDO_NAMESPACE}, "ips": ["<internalapi-static-ips>"]}]'
is only used when run in an OSPdO adoption scenario and is specific to access resources in ospdo env.